source/reference/command/replSetReconfig.txt

```diff
@@ -53,15 +53,25 @@ considerations:
 - A majority of the set's members must be operational for the
   changes to propagate properly.
 
-- This command can cause downtime as the set renegotiates
-  primary-status. Typically this is 10-20 seconds, but could
-  be as long as a minute or more. Therefore, you should attempt
-  to reconfigure only during scheduled maintenance periods.
-
-- In some cases, :dbcommand:`replSetReconfig` forces the current
-  primary to step down, initiating an election for primary among the
-  members of the replica set. When this happens, the primary node will drop
-  all current connections.
+- The :method:`rs.reconfig()` shell method can trigger the current 
+  primary to step down in some situations. When the primary steps down, 
+  it forcibly closes all client connections.
+  
+  If this command steps down the current primary, it also triggers an 
+  :ref:`election <replica-set-elections>` to select a new 
+  :term:`primary`. The election process typically completes 
+  within 2 seconds. See :ref:`replica-set-elections` for more 
```

Comment:
it should complete within 2 seconds once the election is called for, but it can take up to electionTimeoutMillis for the election to start.  Reconfig doesn't actually force an election to happen proactively, it just causes the primary to step down, which will result in an election once the secondaries notice the lack of a primary

Reply:
Made some tweaks to emphasize this.